inc/spbc-settings.php

```diff
@@ -5068,7 +5068,7 @@ function spbc_settings__update_account_email($direct_call = false)
 
     global $spbc;
 
-    $account_email = Post::get('accountEmail');
+    $account_email = Post::get('accountEmail'); // validation is done next line
```

Comment:
Maybe use getString with validation right there? But we need to process new type of validation there:

https://github.com/CleanTalk/wordpress-antispam/blob/7d8f6be4e8e3d5d2f2648dd1ed0b3effe0cae76d/lib/Cleantalk/Common/Validate.php#L17

Then we can use something like this

$account_email = Post::getString('accountEmail', 'email', null );

Reply:
Thanks, fixed https://github.com/CleanTalk/security-malware-firewall/pull/492/commits/6b12e35a1534a8c212021ba02fdf7fa6ca059372

Please recheck